SmartCar/Arduino/sketch_main/sketch_main.ino

```diff
@@ -242,6 +284,77 @@ if(temperatureActivation){
     }   
 }
 
+
+/**
+   Updates the camera servos on each iteration, until a 0 is given.
+   Checks both x and y individually so that they both can be moved at once.
+*/
+void cameraUpdate(int &x, int &y) {
+
+  if (x != 0) { 
+    setCameraServoRelative(camera_servo_x, X_MAX_ANGLE, X_MIN_ANGLE, x, cameraX);
+  }
+  if (y != 0) {
+    setCameraServoRelative(camera_servo_y, Y_MAX_ANGLE, Y_MIN_ANGLE, y, cameraY);
+  }
+
+}
+
```

Comment:
Looks ok, but has to be tested with the car and app before a merge! Test all sensors and modules!